### DISCUSSION TITLE
[Support needed] In-Built Placeholder usage

### DISCUSSION THREAD

**Discussion by RandomAccountt** [Zrips/CMI#7055] [Q&A]
`- '%server_time_HH:mm_bst%'` outputs 22:17 when it's obviously 23:17. 

I'm using the placeholder as stated on the website: `%server_time_[timeFormat]_[timeZone]%`. For context, I'm trying to add UK time to the tablist, but my server is located in a different country and I'm unable to change the servertime. 

I would appreciate someone to inform me of the proper usage for this placeholder, thanks !

**Comment by CloudeLecaw** [comment]
u could shoow your own time too or server time with correct timezone...
here some lessson about it
https://help.gooddata.com/cloudconnect/manual/date-and-time-format.html

so we talk about utc timezones
so it would be
uk is london for utc...
%server_time_HH:mm_+01:00%

you can use UTC and GMT its up to you
